cfme/utils/appliance/__init__.py

```diff
@@ -2636,10 +2637,8 @@ def configure(self, setup_fleece=False, log_callback=None, on_openstack=False, *
 
         # Defer to the IPAppliance.
         super(Appliance, self).configure(log_callback=log_callback, on_openstack=on_openstack)
-        # And do configure the fleecing if requested
-        if setup_fleece:
-            self.configure_fleecing(log_callback=log_callback)
 
+    #  TODO Can we remove this?
```

Comment:
For vsphere this method is provided by the fixtures https://github.com/ManageIQ/integration_tests/blob/0fc21aaf6c6d633bfe52c1d36e9a045e377520e0/cfme/fixtures/appliance.py#L261-L278

Unless we want to keep it as a convenience method for SSA testing. @sbulage do you make use of this method during 
your SSA testing?

Reply:
@john-dupuy No I didn't write it.

Reply:
Thanks for pointing this out @john-dupuy, @jawatts and I discussed it and decided because of the larger assumptions and actions in configure_fleecing, we would leave it in place for now.